src/cascadia/TerminalApp/MinMaxCloseControl.xaml

```diff
@@ -130,7 +130,10 @@
     </StackPanel.Resources>
 
     <Grid x:Name="Content"></Grid>
-    <Border Height="36.0" MinWidth="160.0" x:Name="DragBar" Background="{ThemeResource ApplicationPageBackgroundThemeBrush}" DoubleTapped="DragBar_DoubleTapped"/>
+	<Border Height="36.0"
+					MinWidth="160.0"

```

Comment:
there's a loottt of spaces here. could you align the properties vertically after the space after the tag name?

Reply:
(I think this is a tab/spaces thing)